ocs_ci/ocs/resources/pod.py

```diff
@@ -2333,10 +2333,13 @@ def get_pod_restarts_count(
     restart_dict = {}
     ocp_pod_obj = OCP(kind=constants.POD, namespace=namespace)
     for p in list_of_pods:
-        # we don't want to compare osd-prepare and canary pods as they get created freshly when an osd need to be added.
+        # we don't want to compare osd-prepare and canary pods as they get
+        # created freshly when an osd need to be added. Also skip check on ceph-file-controller-detect-version
+        # as it's a temp. pod.
         if (
             "rook-ceph-osd-prepare" not in p.name
             and "rook-ceph-drain-canary" not in p.name
+            and "ceph-file-controller-detect-version" not in p.name
```

Comment:
imo, this if got too many condition, i would consider something like this:
```
exclude_names = (
    "rook-ceph-osd-prepare",
    "rook-ceph-drain-canary",
    "ceph-file-controller-detect-version"
)

if all(exclude_name not in p.name for exclude_name in exclude_names):
```

Reply:
good idea, updated